src/test/utils/VaultUtils.sol

```diff
@@ -271,6 +311,41 @@ abstract contract BasicVaultTest is UniswapV2Fixture {
         assertEq(getToken0Active(), 0);
     }
 
+    function test_advanceWithNoUserDepositedTokens() public {
+        (uint256 reserves0, uint256 reserves1) = getPairReserves();
+
+        vm.startPrank(address(uint160(strategist) + 1));
+        vm.expectRevert("NOT_PARTICIPANT_OR_STRATEGIST");
+        vault.nextEpoch(reserves0, reserves1);
+    }
+
+    function test_advanceWithDepositedToken1() public {
```

Comment:
let's throw in a `advanceWithDepositedToken0` test case too, just to be safe, because that's what most users have